src/globus_sdk/transport/retry_policy.py

```diff
@@ -0,0 +1,49 @@
+import time
+import typing
+
+import requests
+
+
+class RetryPolicy:
+    max_retries: int = 5
+    max_sleep: int = 10
+
+    def _backoff(self, attempt):
+        # (0.5s * 2^attempt) results in ~15s of total sleep for 5 retries
+        # 2^0 + 2^1 + 2^2 + 2^3 + 2^4 = 31
+        # note that additional time can be spent sending the request or waiting for
+        # a reply
+        time.sleep(0.5 * (2 ** attempt))  # exponential backoff
```

Comment:
Similarly here, having the actual sleep kind of buried I think makes this overall harder to follow.

Reply:
Ach! I don't know how `retry_policy.py` crept back into this during the rebase. I'll remove it entirely. It was renamed to `retry.py`, but somehow rebasing after the `src/` work accidentally reintroduced it. I'll have to do a pass to scrub it.
Please ignore this file. I didn't notice it "reappear".